backend/src/controllers/coordinatorController.js

```
@@ -1,25 +1,88 @@
+    const coordinatorId = req.user.id;
+    const result = await coordinatorService.createDisaster(coordinatorId, req.body);
+    return sendSuccessResponse(res, result, 'Disaster created successfully');
+  } catch (error) {
+    return sendErrorResponse(res, error.message || 'Internal Server Error', error.statusCode || 500);
+  }
+};
+
+const getDisasters = async (req, res) => {
+  try {
+    const { page = 1, limit = 10 } = req.query;
+    const offset = (parseInt(page) - 1) * parseInt(limit);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)